### PR TITLE
Doc: Add known issue and fixed issue in release note

### PR DESCRIPTION
**Pull request by fuzhongl**
Signed-off-by: fuzhongl <fuzhong.liu@eintel.com>

### CODE REVIEW

File: doc/release_notes/release_notes_2.6.rst [hunk 1]
```diff
@@ -80,8 +80,38 @@ Fixed Issues Details
 
 .. comment example item
    - :acrn-issue:`5626` - [CFL][industry] Host Call Trace once detected
-
+- :acrn-issue: '6012' -  [Mainline][PTCM] [ConfigTool]Obsolete terms cleanup for SSRAM
```
Comment (dbkinder) [inline doc/release_notes/release_notes_2.6.rst:83]: The format of these issues is wrong (see the comment)

```suggestion
- :acrn-issue:`6012` -  [Mainline][PTCM] [ConfigTool]Obsolete terms cleanup for SSRAM
```

File: doc/release_notes/release_notes_2.6.rst [hunk 2]
```diff
@@ -80,8 +80,38 @@ Fixed Issues Details
 
 .. comment example item
    - :acrn-issue:`5626` - [CFL][industry] Host Call Trace once detected
-
```
Comment (dbkinder) [inline doc/release_notes/release_notes_2.6.rst:83]: This blank line after the comment is needed

**Review by dbkinder — CHANGES_REQUESTED** [review]

File: doc/release_notes/release_notes_2.6.rst (hunk 1 above)
Comment (fuzhongl) [reply doc/release_notes/release_notes_2.6.rst]: @dbkinder Thanks!
Done!

File: doc/release_notes/release_notes_2.6.rst (hunk 2 above)
Comment (fuzhongl) [reply doc/release_notes/release_notes_2.6.rst:83]: @dbkinder Done!
Thanks!

File: doc/release_notes/release_notes_2.6.rst [hunk 3]
```diff
@@ -80,8 +80,37 @@ Fixed Issues Details
...
-
+- :acrn-issue: `6012` -  [Mainline][PTCM] [ConfigTool]Obsolete terms cleanup for SSRAM
```
[the remark this answers is not in the document]
Comment (fuzhongl) [reply doc/release_notes/release_notes_2.6.rst]: Done!

**Review by dbkinder — APPROVED** [review]

**Pull request merged**